src/docfx/config/ops/OpsConfig.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.Docs.Build
     [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
     internal class OpsConfig
     {
+        // TODO: remove this switch
```

Comment:
Add the task link here? We can better know when we can remove this.

https://ceapex.visualstudio.com/Engineering/_workitems/edit/230323/

Reply:
Prefer to revert this PR when the work item is ready